### PR TITLE
Russian translation fixes/improvements

### PR DESCRIPTION
**Pull request by Blueberryy**

### CODE REVIEW

**Comment by jekyllgrim** [comment]
While this request contains some useful changes, there are some issues that do not allow me to merge it as is. For example:
* "Сюрикэн" is the correct accepted spelling, not "сюрикен" (please check any publicly available source).
* Capitalization of key terms, such as "Кодекс" is intentional and does not need to be changed.
* There are factual mistakes in the suggested changes. For example, about the flamethrower: "огонь летит с большей скоростью, повышая радиус поражения" — this is  not "радиус поражения" (damage radius) but rather the effective distance (the distance the projectiles can cover), so this suggestion is incorrect.
* There are some unnecessary suggestions as well. For example, the current line "не оставляют после себя душ" does not need to be corrected, it's already grammatically valid (see the rules regarding cases after negative verbs, e.g. [here](https://www.textologia.ru/russkiy/morfologia/imya-sushestvitelnoe/padezh-pri-otricanii-roditelniy-ili-vinitelniy-dolzhen-bit-padezh-suschestvitelnogo-pri-glagolah-s-otricaniem/6985/?q=463&n=6985)).
* The use of letter "ё" is optional in Russian aside from the cases where it can cause confusion (such as "все/всё"). I agree that it's worth implementing it, though.

In short, I can't accept this commit as is, but I'm prepared to add some of these improvements later myself.

**Comment by Blueberryy** [comment]
> While this request contains some useful changes, there are some issues that do not allow me to merge it as is. For example:
> 
> * "Сюрикэн" is the correct accepted spelling, not "сюрикен" (please check any publicly available source).
> * Capitalization of key terms, such as "Кодекс" is intentional and does not need to be changed.
> * There are factual mistakes in the suggested changes. For example, about the flamethrower: "огонь летит с большей скоростью, повышая радиус поражения" — this is  not "радиус поражения" (damage radius) but rather the effective distance (the distance the projectiles can cover), so this suggestion is incorrect.
> * There are some unnecessary suggestions as well. For example, the current line "не оставляют после себя душ" does not need to be corrected, it's already grammatically valid (see the rules regarding cases after negative verbs, e.g. [here](https://www.textologia.ru/russkiy/morfologia/imya-sushestvitelnoe/padezh-pri-otricanii-roditelniy-ili-vinitelniy-dolzhen-bit-padezh-suschestvitelnogo-pri-glagolah-s-otricaniem/6985/?q=463&n=6985)).
> * The use of letter "ё" is optional in Russian aside from the cases where it can cause confusion (such as "все/всё"). I agree that it's worth implementing it, though.
> 
> In short, I can't accept this commit as is, but I'm prepared to add some of these improvements later myself.

Okay good luck on that

**Comment by jekyllgrim** [comment]
Partially implemented in 812977c9a6293c904fa785768a2d1aad462f986f and will be included in the 1.3.0 release (soon). Thanks! Sorry it took so long.
